go.sum

```diff
@@ -79,6 +81,8 @@ github.com/newrelic-forks/newrelic-telemetry-sdk-go v0.2.1-0.20200423081332-b685
 github.com/newrelic-forks/newrelic-telemetry-sdk-go v0.2.1-0.20200423081332-b6856fd89c75/go.mod h1:walb6Ok1czNkvTIJQrdXF9NpthAPCwCFu/gd2MxoPVY=
```

Comment:
I guess these are still being used until headers work gets merged, right?

Reply:
I thought I got rid of that. Let me check why it is still there.

Reply:
I cleaned up the `go.sum` file so no references to that package any more.